app/controllers/analytics_controller.rb

```diff
@@ -0,0 +1,22 @@
+class AnalyticsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  before_action :confirm_two_factor_authenticated
+
+  def create
```

Comment:
This is a bit tough for me to follow. The `AnalyticsController` is being used to track platform authenticators? Wouldn't it make more sense for this to be a `PatformAuthenticatorsController` under the `Analytics` namespace?

Reply:
I thought about that but then decided against it because all the analytics in javascript will be gathered in one call.  You don't want multiple calls pinging the analytics to the backend.

Reply:
Right now it's not built like that though, right? There's one param it accepts, "available" and it refers to platform authenticators.

Reply:
True.  I didn't make the format very extensible.

Reply:
I do agree with Jim that we shouldn't trust the client on the `success` attribute. Adding the value to `extra` make more sense to me.

My concern here is more related to the fact that the API we are building doesn't have the attributes we are passing namespaced, which it seems like it should. I'd think we want the client to send something like this:

```javascript
{
  platform_authenticator: {  available: true }
}
```

Reply:
I disagree.  We can always change up the javascript/interface later.  But I do agree that the format that we log should be set now.

Reply:
We can, but it will be more difficult then. It will take at least 2 releases. One to support both formats and an additional one to drop the old format.

Reply:
I already changed the format.  See the updated screenshot.